src/Raven.Client/Extensions/SecurityExtensions.cs

```diff
@@ -25,7 +25,7 @@ internal static void InitializeSecurity(ConventionBase conventions, HttpJsonRequ
             {
                 if (credentials.ApiKey == null)
                 {
-                    AssertForbiddenCredentialSupportWindowsAuth(forbiddenResponse, primaryCredentials);
+                    // AssertForbiddenCredentialSupportWindowsAuth(forbiddenResponse, primaryCredentials);
```

Comment:
Need to remove this entirely, not just comment, and then test that we get the appropriate error from the client on invalid access